std/file.d

```diff
@@ -237,14 +237,14 @@ void[] read(R)(R name, size_t upTo = size_t.max)
     import std.utf : byChar;
     scope(exit)
     {
-        assert(exists("someUniqueFilename"));
-        remove("someUniqueFilename");
+        assert(exists(deleteme));
+        remove(deleteme);
```

Comment:
[@burner had this right the first time](https://github.com/dlang/phobos/pull/3643#discussion-diff-39070724), @quickfur, following what the rest of this module does.  All phobos tests that create a file use `deleteme`, because it leads to all test files being in `tempDir()` rather than the local directory and is easy to use with Android.  If having this in the docs is a concern, I can always modify it to add a name you prefer at the end, but considering we are deleting the file, I don't see the problem.